game_eggs/steamcmd_servers/v_rising/egg-v-rising.json

```diff
@@ -0,0 +1,231 @@
+            "default_value": "V Rising Dedicated Server",
+            "user_viewable": true,
+            "user_editable": true,
+            "rules": "required|string"
+        },
+        {
+            "name": "Server Description",
+            "description": "Short description of server purpose, rules, and the message of the day.",
+            "env_variable": "DESCRIPTION",
+            "default_value": "Welcome to the server!",
+            "user_viewable": true,
+            "user_editable": true,
+            "rules": "nullable|string"
+        },
+        {
+            "name": "Max Connected Users",
+            "description": "Max number of concurrent players on the server.",
+            "env_variable": "MAX_PLAYERS",
+            "default_value": "40",
+            "user_viewable": true,
+            "user_editable": true,
+            "rules": "required|integer|min:1"
+        },
+        {
+            "name": "Max Connected Admins",
+            "description": "Max number of admins to allow connection even when the server is full.",
+            "env_variable": "MAX_ADMINS",
+            "default_value": "4",
+            "user_viewable": true,
+            "user_editable": true,
+            "rules": "required|integer|min:0"
+        },
+        {
+            "name": "Server Password",
+            "description": "Password required to join the server. Leave blank to have no password.",
+            "env_variable": "SERVER_PASSWORD",
+            "default_value": "",
+            "user_viewable": true,
+            "user_editable": true,
+            "rules": "nullable|string"
+        },
+        {
+            "name": "Save Name",
+            "description": "Name of save file \/ directory.",
+            "env_variable": "SAVE_NAME",
+            "default_value": "world1",
+            "user_viewable": true,
+            "user_editable": true,
+            "rules": "required|string"
+        },
+        {
+            "name": "Auto Save Count",
+            "description": "Number of autosaves to keep.",
+            "env_variable": "SAVE_COUNT",
+            "default_value": "50",
+            "user_viewable": true,
+            "user_editable": true,
+            "rules": "required|integer|min:0"
+        },
+        {
+            "name": "Auto Save Interval",
+            "description": "Interval in seconds between each autosave.",
+            "env_variable": "SAVE_INTERVAL",
+            "default_value": "600",
+            "user_viewable": true,
+            "user_editable": true,
+            "rules": "required|integer|min:1"
+        },
+        {
+            "name": "List On Master Server",
+            "description": "Set to true to list on the server list, else set to false.",
+            "env_variable": "SERVER_LIST",
+            "default_value": "true",
+            "user_viewable": true,
+            "user_editable": true,
+            "rules": "required|string|in:true,false"
```

Comment:
This will save true in a wrong way.

it should be:

`"ListOnMasterServer": true,`

and with this it is

"ListOnMasterServer": "true",


Then the config don't work

Reply:
`"true"` and `"false"` happen to work for the RCON setting in the same file (I tested it), so I assume the JSON parser in this particular server application is okay with string booleans (unlike Arma Reforger, for example). Do you know for certain that it will not register properly for the `ListOnMasterServer` setting specifically? It's hard to test this setting reliably given the reports that servers sometimes don't show on the server in general.

Reply:
thats the problem in my VRising egg and thats why i open  https://github.com/pterodactyl/panel/issues/4087
with "true", true is not recocnized.

"Setting" true, -> ok
"Setting" "true", not possible

Reply:
Why does it work for the RCON setting then? It wouldn't make sense for different settings in the file to be parsed differently. Have you considered the possibility that your server isn't in the serverlist due to a factor other than the config file (like server popularity, or master server lag)? Also, the default value for that setting is true according to documentation, so even if it thinks `"true"` is invalid, it would default to `true` anyways.

Reply:
Worst case scenario I can change it to file parser, but it seems to be working for me for this specific server application.

Reply:
> Worst case scenario I can change it to file parser, but it seems to be working for me for this specific server application.

I used the same rule before the last patch and it definitely didn't work. There is also a conversation on their Discord somewhere, because I had asked why it does not work with "VALUE", but only with VALUE.

It seems as if they have fixed this with the last patch quietly :)

"ListOnMasterServer": "true", -> works

RCON "true", -> works
Secure "true",-> works